### PR TITLE
feat-moduleui(Intial Design done)

### PR DESCRIPTION
**Pull request by sirichandana1018**
Please Merge!

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -1,81 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
```
Comment (kashyapmukkamala) [inline .idea/workspace.xml]: add `./idea` to `.gitignore` file

**Comment by sirichandana1018** [comment]
Could you check now?

File: src/css/index.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
+body{
+    background-color: #0d0d0d;
+}
+
+.navbar-inverse .navbar-brand {
+    color:#E50914;
+    font-family: serif;
+    font-weight: 600;
+    font-size: 32px;
```
Comment (kashyapmukkamala) [inline src/css/index.css]: do not hardcode sizes

File: src/css/index.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+.firstimg h2{
+    position: absolute;
+    top: 200px;
+    left: 25px;
+    width:100%;
+    color:white;
+}
+.firstimg h3{
+    position: absolute;
+    top: 255px;
```
Comment (kashyapmukkamala) [inline src/css/index.css]: do not hardcode

File: src/css/index.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.firstimg p{
+    position: absolute;
+    top: 325px;
+    color:white;
+    left: 25px;
+    width:40%;
+    font-weight: 400;
+    font-size: 1.4vw;
```
Comment (kashyapmukkamala) [inline src/css/index.css]: only use `em`, `rem` and `%` for font size

File: src/index.html [hunk 5]
```diff
@@ -1,40 +1,357 @@
...
+    <p> Evolving from single-celled yellow organisms at the dawn of time, Minions live to serve,
+        but find themselves working for a continual series of unsuccessful masters. </p>
+</div>
+
+<div class="container1">
+    <p>
+        Continue Watching for Siri
+    </p>
+    <div class="col-md-12">
+        <img src="http://wac.450f.edgecastcdn.net/80450F/b985.fm/files/2016/01/Angry-Birds.jpg?w=300&h=200&zc=1&s=0&a=t&q=89" alt="CSS3">
```
Comment (kashyapmukkamala) [inline src/index.html]: why is alt `CSS3`?

**Pull request merged**